src/functions/postTypes/processPostTypeQuery.js

```diff
@@ -83,18 +86,24 @@ export default async function processPostTypeQuery(
       }
 
       // Handle blocks.
-      newPost.blocks = await formatBlockData(
-        JSON.parse(newPost.blocksJSON) ?? []
+      const { blocksJSON, commonBlockJSON } = newPost;
+
+      const blocks = await formatBlockData(JSON.parse(blocksJSON) ?? []);
+
+      const commonPostBlocks = await formatBlockData(
```

Comment:
What is in blocks/commonPostBlocks?

Reply:
@kilemensi commonPostBlocks are/will be the blocks in the parent Stories page. (i.e common to all post/story pages but added only once on the parent) Currently we have share props (social icons) that appear on every post, and the hero that appear on `stories/news` & `stories/isnights`

Reply:
Why do the come from `homepageSettings` then?

Reply:
@kilemensi that's just bad naming and because we like copy pasting speak_no_evil We are registering a homepageSettings object field here that pulls the wordpress writing settings ( from wp_options data). On the writing settings, is where we define our homepage and our main posts page. homePageSettings should be & can be named differently